docs/dmytro-shevchenko1/GamesProject/styles/cards.css

```diff
@@ -0,0 +1,245 @@
+.filter {
+  background-color: #221F1F;
+  padding: 12px 53px;
+  font-size: 14px;
+  font-weight: 400;
+  color: #FFF;
+}
+
+.filter-form {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.filter-search {
+  display: flex;
+}
+
+.search-input {
+  margin-right: 0;
+  border-top-left-radius: 6px;
+  border-bottom-left-radius: 6px;
+  padding: 12px 100px 12px 45px;
+  background-image: url("../images/search_logo.svg");
+  background-repeat: no-repeat;
+  background-position: 10px 7px;
+  border: #FFF;
+  outline: none;
+}
+
+.search-button {
+  border-top-right-radius: 6px;
+  border-bottom-right-radius: 6px;
+  margin-left: 0;
+  color: #FFF;
+  background-color: #EF4934;
+  border: none;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.select-name {
+  border-radius: 6px;
+  padding: 13px;
+  background-color: #333;
+  color: #FFF;
+  outline: none;
+}
+
+.select-name option {
+  background-color: white;
+  color: #221F1F;
+}
+
+.cards-main {
+  padding: 0 53px;
+  margin-bottom: 46px;
+}
+
+.cards-h1 {
+  color: #221F1F;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  margin: 40px 0;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 25px;
+  padding: 0;
+}
+
+ul.cards {
```

Comment:
Why do you need it? You can move `margin: 0;` into `.cards` selector, on line 68